Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
 [workspace.package]
 edition = "2021"
 # Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml and with the relevant README.md files.
+# TODO: When this hits 1.74, move lint configuration into this file via a lints table.
```

Comment:
We don't need to wait for the MSRV to be that high to use the lints table. The MSRV `cargo check` commands won't fail, they'll just warn that there is an unused manifest key but the CI will still pass. The stable toolchain will do the actual linting.

This won't be a problem for projects higher up the stack either, as cargo doesn't do linting of dependencies.